src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,88 @@
+package seedu.addressbook.ui;
+
+import java.io.PrintStream;
+import java.util.List;
+
+/**
+ * Does all formatting and styling of output text for the application.
+ */
+
+public class Formatter {
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+    
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    private final PrintStream _out;
+    
+    public Formatter(PrintStream out) {
+    	_out = out;
+    }
+    
+    /**
+     * Prints a string to the print stream (without new line)
+     */
+    public void print(String string) {
+    	_out.print(LINE_PREFIX + string.replace("\n", LS + LINE_PREFIX));
+    }
+
+    /**
+     * Prints a blank line to the print stream
+     */
+    public void printBlankLine() {
+    	printLines("");
+    }
+
+    /**
+     * Prints multiple lines to the print stream
+     */
+	public void printLines(String... strings) {
+		for (String string : strings) {
+			print(string);
+			_out.println();
+		}
+	}
+
+    /**
+     * Prints a divider of 1 line
+     */
+	public void printDivider() {
+		printDivider(1);
+	}
+	
+    /**
+     * Prints a divider of {@code height} lines
+     */
+	public void printDivider(int height) {
+		for (int i = 0; i < height; i ++) {
+			printLines(DIVIDER);
+		}
+	}
+	
+	/** Prints a list of strings as a viewable indexed list. */
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.